packages/blade/src/components/Popover/_decisions/decisions.md

```diff
@@ -0,0 +1,275 @@
+| defaultIsOpen   | `boolean`                                                                                                              | `undefined` | Uncontrolled state of the popover                                                                                                     |          |
+| onOpenChange    | `({ isOpen }) => void`                                                                                                 | `undefined` | Called when popover isOpen state is changed, this can be used to detect when popover opens or closed                                  |          |
+| initialFocusRef | `React.RefObject`                                                                                                      | `undefined` | The ref of the element that should receive focus when the popover opens.                                                              |          |  |
+
+### Examples:
+
+#### Usage
+
+```js
+// basic example
+<Popover
+  placement="top-end"
+  headerTitle="International Payments"
+  content={
+    <Text>
+      Your business can go international with support for transactions in 
+      <Link href="#">100 foreign currencies.</Link>
+    </Text>
+  }
+>
+  <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+</Popover>
+```
+
+<img src="./assets/popover-example-1.png" alt="Example usage of popover" width="70%" />
+
+#### Controlled 
+
+```js
+const ControlledPopover = () => {
+  const [isOpen, setIsOpen] = React.useState(false);
+
+  return (
+    <>
+    <Button onClick={()=> setIsOpen(prev => !prev)}>Trigger</Button>
+    <Popover
+      isOpen={isOpen}
+      onOpenChange={({ isOpen }) => setIsOpen(isOpen)}
+      content={<Text>Any content</Text>}
+    >
+      <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+    </Popover>
+    </> 
+  )
+}
+```
+
+#### Uncontrolled 
+
+```js
+const UncontrolledPopover = () => {
+  return (
+    <Popover
+      defaultIsOpen={true}
+      content={<Text>Any content</Text>}
+    >
+      <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+    </Popover>
+  )
+}
+```
+
+#### On initial load 
+
+To open the popover on initial load, consumers can just pass `defaultIsOpen` prop and whenever the popover mounts it will open.
+
+```js
+const InitialOpen = () => {
+  return (
+    <Popover
+      defaultIsOpen={true}
+      content={<Text>Any content</Text>}
+    >
+      <IconButton icon={InfoIcon} accessibilityLabel="Refund" />
+    </Popover>
+  )
+}
+```
+
+#### Working with non-interactive triggers
+
+To make popover apear on clicking a non-interactive elements such as icons, badges, counters etc. We will provide a `PopoverInteractiveWrapper` component which will work as a minimal trigger:
+
+```js
+import { Popover, PopoverInteractiveWrapper } from '@razorpay/blade';
+
+// non-interactive element as trigger
+<Popover content="Amount reversed to customer bank account">
+  <PopoverInteractiveWrapper>
```

Comment:
is there anything special in `PopoverInteractiveWrapper` compared to the tooltip one? if not can we use the same wrapper here as well?